recipes/hatchling/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "hatchling" %}
+{% set version = "0.15.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hatchling-{{ version }}.tar.gz
+  sha256: 33816429b78d0f069717ccce53a2ec722c79412c86928caa2e01a93b4eaf31a5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - hatchling = hatchling.cli:hatchling
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
```

Comment:
If toml is required for python < 3.6, I think I'd just use >3.6 here. Conda-forge doesn't build for anything less than 3.7 anymore anyway.